Makefile

```diff
@@ -98,16 +108,27 @@ install-scripts:
 	    install -p -m 755 $$script $(AGENTEXECDIR); \
 	done
 
+.PHONY: install
 install-app: install-app-selinux install-app-config install-scripts
 
-install: install-app
+.PHONY: install
+install: install-app ## Install rabe-zabbix to $(PREFIX)
```

Comment:
It would be helpful, if you add a short description of what exactly _rabe-zabbix_ contains. For example: `Install rabe-zabbix (user parameters, helper scripts and SELinux policies) to $(PREFIX)`

Reply:
@hairmare: I would like to merge this PR, any chance that you will find the time to react on my review comment above? Just let me know if you don't agree with my proposal, it's certainly not a blocker, but should (in my opinion) help other users understand what's going on behind the scenes. Thanks!

Reply:
I'd rather keep this minimal since I don't want to have to update the Makefile docstrings when we introduce new features. We should probably add more docs on the Makefile to the docs at some point. I'd prefer to let it settle featurewise before we do that.